crates/oci/src/client.rs

```diff
@@ -688,6 +704,68 @@ fn registry_from_input(server: impl AsRef<str>) -> String {
     }
 }
 
+fn infer_annotations(
+    locked_app: &LockedApp,
+    explicit: Option<BTreeMap<String, String>>,
+    predefined: PredefinedAnnotations,
+) -> Option<BTreeMap<String, String>> {
+    use spin_locked_app::{MetadataKey, APP_DESCRIPTION_KEY, APP_NAME_KEY, APP_VERSION_KEY};
+    const APP_AUTHORS_KEY: MetadataKey<Vec<String>> = MetadataKey::new("authors");
+
+    if predefined == PredefinedAnnotations::Skip {
+        return explicit;
+    }
+
+    // We will always, at minimum, have a `created` annotation, so if we don't already have an
+    // anootations collection then we may as well create one now...
+    let mut current = explicit.unwrap_or_default();
+
+    let authors = locked_app
+        .get_metadata(APP_AUTHORS_KEY)
+        .unwrap_or_default()
+        .unwrap_or_default();
+    if !authors.is_empty() {
+        let authors = authors.join(", ");
+        add_inferred(
+            &mut current,
+            "org.opencontainers.image.authors",
```

Comment:
We should be able to utilize the [upstream definitions](https://github.com/oras-project/rust-oci-client/blob/main/src/annotations.rs) for these keys, via our current fermyon/oci-distribution dependency.

Reply:
EGAD

I didn't even think to look!  Thanks @vdice